test/unit/gateways/cyber_source_test.rb

```diff
@@ -849,10 +849,33 @@ def test_adds_application_id_as_partner_solution_id
     end.check_request do |endpoint, data, headers|
       assert_match("<partnerSolutionID>#{partner_id}</partnerSolutionID>", data)
     end.respond_with(successful_capture_response)
-  ensure
     CyberSourceGateway.application_id = nil
   end
 
+  def test_missing_field
+    @gateway.expects(:ssl_post).returns(missing_field_response)
+
+    response = stub_comms do
+      @gateway.purchase(@amount, credit_card, @options)
+    end.check_request do |_endpoint, body, _headers|
+      assert_xml_valid_to_xsd(body)
+      assert_match %r'<c:missingField>c:billTo/c:country</c:missingField>'m, body
+    end.respond_with(missing_field_response)
+    assert_failure response
+  end
+
+  def test_invalid_field
+    @gateway.expects(:ssl_post).returns(invalid_field_response)
+
+    response = stub_comms do
+      @gateway.purchase(@amount, credit_card, @options)
+    end.check_request do |_endpoint, body, _headers|
+      assert_xml_valid_to_xsd(body)
+      assert_match %r'<c:invalidField>c:billTo/c:postalCode</c:invalidField><'m, body
+    end.respond_with(missing_field_response)
```

Comment:
Should this be 
```suggestion
    end.respond_with(invalid_field_response)
```
?

Reply:
Yes! Good catch.

Reply:
https://github.com/activemerchant/active_merchant/blob/277b123d1f3da81464a6515c355635923d6a4805/test/unit/gateways/cyber_source_test.rb#L876